x-pack/plugins/observability_solution/synthetics/server/routes/settings/private_locations/get_private_locations.ts

```diff
@@ -52,21 +52,24 @@ export const getPrivateLocationsRoute: SyntheticsRestApiRouteFactory<
 
 export const getPrivateLocationsAndAgentPolicies = async (
   savedObjectsClient: SavedObjectsClientContract,
-  syntheticsMonitorClient: SyntheticsMonitorClient
+  syntheticsMonitorClient: SyntheticsMonitorClient,
+  excludeAgentPolicies: boolean = false
 ): Promise<SyntheticsPrivateLocationsAttributes & { agentPolicies: AgentPolicyInfo[] }> => {
   try {
     const [privateLocations, agentPolicies] = await Promise.all([
       getPrivateLocations(savedObjectsClient),
-      syntheticsMonitorClient.privateLocationAPI.getAgentPolicies(),
+      excludeAgentPolicies
+        ? new Promise<void>((resolve) => resolve())
+        : syntheticsMonitorClient.privateLocationAPI.getAgentPolicies(),
```

Comment:
i think this can be removed al-together this isn't being used anymore , anywhere.

Reply:
We discussed offline it is, in fact, being used.